### DISCUSSION TITLE
Add alias for 'dotnet-format'

### DISCUSSION THREAD

**Discussion by ap0llo** [cake-build/cake#3463] [Extension requests]
Hi, I would like to run [`dotnet-format`](https://github.com/dotnet/format) from my Cake scripts and thus add an alias to make calling it easier.

Would this be something you waould want to include in [Cake.Common along with the other aliases](https://github.com/cake-build/cake/tree/develop/src/Cake.Common/Tools) (I'd be happy to provide a PR) or should this instead be a separate addin?

**Comment by gep13** [comment]
Personally, I think this would be better served as an addin, rather than directly into Cake.Common.

We could create a repository in the cake-contrib Organization if you wanted to do the development work over there.

**Reply by augustoproiete** [reply]
+1. Starting as an addin is a good way to get something useful out quickly and test it out. If the addin becomes widely successful & widely used it can be brought into the main codebase in the future.

**Reply by ap0llo** [reply]
Okay, thanks for the guidance - I'll start implementing it as an addin.

Once I've got something working, I'll consider transferring it to cake-contrib
